wiki/Performance_Points/en.md

```diff
@@ -1,156 +1,130 @@
-<img src="Performance point.jpg" title="fig:Performance point and Performance rank" alt="Performance point and Performance rank" width="200" /> Performance Points (pp) is a ranking metric aiming to be more contextually relevant to a continuous game like osu!. It aims to shift the focus from being on the amount of time played to the **actual skill of the player**.
+Performance Points
+==================
+
+Performance Points (pp) is a ranking metric that aims to be more contextually relevant to a players in a continuous game like *osu!*.
+
+It aims to shift the focus of skill progression from the amount of time played to an **actual representation of the player's skill.**
 
 History
 =======
 
-The new metric was initially released in April, 2012 as '???' during a testing phase, being finally renamed to "pp" (Performance Points) on the 17th April 2012. In July 24, the [(20120722-24) osu! release](http://osu.ppy.sh/forum/p/1687719) replaced the Ranked score system with Performance Points for every 30 minutes. As of August 16, the pp system is now updated in real-time. After more than a year of service, ppv1 received revamped upgrade (named ppv2), which was implemented on the 27th January 2014 by [Tom94](http://osu.ppy.sh/u/1857058), creator of the [osu!tp ranking system](http://osutp.net).
+Revealed to the public during April 2012 as the mysterious *'???'* testing phase, the system was eventually renamed to "pp" later on in the month. Several months after its reveal, the [(20120722-24) osu! release](http://osu.ppy.sh/forum/p/1687719) officially implemented the system to fully replace the old Ranked score system, with new scores being calculated every 30 minutes. Later on in August in the same year, the system was improved to update in real-time. 
+
+It continued to exist in this capacity for more than a year of service, until [Tom94](http://osu.ppy.sh/u/1857058), the creator of the *osu!tp* scoring metric, joined the osu! team and implemented his design into the system. The resulting system was titled *ppv2*, and became live on the 27th of January, 2014.
+
+*ppv2* is currently in active service, with live updates published to its [changelog](https://osu.ppy.sh/p/changelog?category=pp).
+
+*ppv1*, the original build of the Performance Points system also had a changelog, which you may view from its [forum topic](http://osu.ppy.sh/forum/t/92185).
 
-A changelog reflecting the most recent changes to the system is available [here](https://osu.ppy.sh/p/changelog?category=pp). For the obsolete ppv1 changelog, look [here](http://osu.ppy.sh/forum/t/92185).
+How is pp calculated?
+=====================
 
-Location
-========
+Performance points are very heavily based on calculated map difficulty, which is determined by a unique algorithm constructed for each individual gamemode. 
```

Comment:
why the sudden change of cases with Performance Points?
`Performance points are very` ->
`Performance Points are very`